brat/Brat/Checker.hs

```diff
@@ -419,6 +487,73 @@ check' (Simple tm) ((), ((hungry, ty):unders)) = do
       pure (((), ()), ((), unders))
 check' tm _ = error $ "check' " ++ show tm
 
+
+-- Clauses from either function definitions or case statements (TODO), as we get
```

Comment:
Think this TODO is bogus now? The TODO was this PR! :)

Reply:
Fixed!